### PR TITLE
Prepare everything for sync-async.

### PR DESCRIPTION
**Pull request by floitschG**
Prepares Flutter for a VM roll where sync-async is enabled by default.

### CODE REVIEW

File: packages/flutter_test/lib/src/binding.dart [hunk 1]
```diff
@@ -507,6 +507,12 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
   }
 
   Future<Null> _runTestBody(Future<Null> testBody(), VoidCallback invariantTester) async {
+    // Delay this function by a microtask.
+    // Otherwise it will open as cope immediately, which then open the
+    // `asyncBarrier` is invoked. The `asyncBarrier` is immediately following
+    // the call to `testZone.runBinary(_runTestBody`, so delaying by one
```
Comment (mravn-google) [inline packages/flutter_test/lib/src/binding.dart]: Add missing `)`

File: packages/flutter_test/lib/src/binding.dart [hunk 2]
```diff
@@ -507,6 +507,12 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
   }
 
   Future<Null> _runTestBody(Future<Null> testBody(), VoidCallback invariantTester) async {
+    // Delay this function by a microtask.
+    // Otherwise it will open as cope immediately, which then open the
```
Comment (mravn-google) [inline packages/flutter_test/lib/src/binding.dart]: I cannot parse this line.

**Review by mravn-google — APPROVED** [review]
LGTM

**Comment by mravn-google** [comment]
@floitschG Can we close this one now?

**Comment by mit-mit** [comment]
@mravn-google do we still need this PR?

**Pull request closed without merging**